projects/kitti_step_dataset/docs/en/dataset_prepare.md

```diff
@@ -0,0 +1,663 @@
+│   │   │   ├── test.txt
+│   │   │   ├── all.txt
+│   ├── coco_stuff164k
+│   │   ├── images
+│   │   │   ├── train2017
+│   │   │   ├── val2017
+│   │   ├── annotations
+│   │   │   ├── train2017
+│   │   │   ├── val2017
+│   ├── CHASE_DB1
+│   │   ├── images
+│   │   │   ├── training
+│   │   │   ├── validation
+│   │   ├── annotations
+│   │   │   ├── training
+│   │   │   ├── validation
+│   ├── DRIVE
+│   │   ├── images
+│   │   │   ├── training
+│   │   │   ├── validation
+│   │   ├── annotations
+│   │   │   ├── training
+│   │   │   ├── validation
+│   ├── HRF
+│   │   ├── images
+│   │   │   ├── training
+│   │   │   ├── validation
+│   │   ├── annotations
+│   │   │   ├── training
+│   │   │   ├── validation
+│   ├── STARE
+│   │   ├── images
+│   │   │   ├── training
+│   │   │   ├── validation
+│   │   ├── annotations
+│   │   │   ├── training
+│   │   │   ├── validation
+|   ├── dark_zurich
+|   │   ├── gps
+|   │   │   ├── val
+|   │   │   └── val_ref
+|   │   ├── gt
+|   │   │   └── val
+|   │   ├── LICENSE.txt
+|   │   ├── lists_file_names
+|   │   │   ├── val_filenames.txt
+|   │   │   └── val_ref_filenames.txt
+|   │   ├── README.md
+|   │   └── rgb_anon
+|   │   |   ├── val
+|   │   |   └── val_ref
+|   ├── NighttimeDrivingTest
+|   |   ├── gtCoarse_daytime_trainvaltest
+|   |   │   └── test
+|   |   │       └── night
+|   |   └── leftImg8bit
+|   |   |   └── test
+|   |   |       └── night
+│   ├── loveDA
+│   │   ├── img_dir
+│   │   │   ├── train
+│   │   │   ├── val
+│   │   │   ├── test
+│   │   ├── ann_dir
+│   │   │   ├── train
+│   │   │   ├── val
+│   ├── potsdam
+│   │   ├── img_dir
+│   │   │   ├── train
+│   │   │   ├── val
+│   │   ├── ann_dir
+│   │   │   ├── train
+│   │   │   ├── val
+│   ├── vaihingen
+│   │   ├── img_dir
+│   │   │   ├── train
+│   │   │   ├── val
+│   │   ├── ann_dir
+│   │   │   ├── train
+│   │   │   ├── val
+│   ├── iSAID
+│   │   ├── img_dir
+│   │   │   ├── train
+│   │   │   ├── val
+│   │   │   ├── test
+│   │   ├── ann_dir
+│   │   │   ├── train
+│   │   │   ├── val
+│   ├── occlusion-aware-face-dataset
+│   │   ├── train.txt
+│   │   ├── NatOcc_hand_sot
+│   │   │   ├── img
+│   │   │   ├── mask
+│   │   ├── NatOcc_object
+│   │   │   ├── img
+│   │   │   ├── mask
+│   │   ├── RandOcc
+│   │   │   ├── img
+│   │   │   ├── mask
+│   │   ├── RealOcc
+│   │   │   ├── img
+│   │   │   ├── mask
+│   │   │   ├── split
+│   ├── ImageNetS
+│   │   ├── ImageNetS919
+│   │   │   ├── train-semi
+│   │   │   ├── train-semi-segmentation
+│   │   │   ├── validation
+│   │   │   ├── validation-segmentation
+│   │   │   ├── test
+│   │   ├── ImageNetS300
+│   │   │   ├── train-semi
+│   │   │   ├── train-semi-segmentation
+│   │   │   ├── validation
+│   │   │   ├── validation-segmentation
+│   │   │   ├── test
+│   │   ├── ImageNetS50
+│   │   │   ├── train-semi
+│   │   │   ├── train-semi-segmentation
+│   │   │   ├── validation
+│   │   │   ├── validation-segmentation
+│   │   │   ├── test
```

Comment:
I think don't need this file and we might move the preparation steps to README.md.